hw/dv/tools/dvsim/vcs.hjson

```diff
@@ -31,7 +31,9 @@
                // otherwise. The double-escaped quotes are because this needs to go inside a string
                // that gets passed to Make (stripping one level of quotes), and then needs to
                // result in an argument with an embedded space (the other one).
-               "-CFLAGS \\\"--std=c99 -fno-extended-identifiers\\\"",
+               "-CFLAGS \\\"--std=c++11 -fno-extended-identifiers\\\"",
+               // This is required for dpi_memutil.cc
+               "-CFLAGS --std=c++11",
```

Comment:
Probably replace the comment above with something like 'C++11 standard is enforced for all sources written in C++, including the DPI-C constructs.'